input/pagecontent/changes.md

```diff
@@ -87,6 +87,7 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from Procedure.bodySite and added profile-specific implementation guidance on including body site information [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017), [FHIR-45114](https://jira.hl7.org/browse/FHIR-45114)
   - removed Must Support from Procedure.note [FHIR-45114](https://jira.hl7.org/browse/FHIR-45114)
   - removed the cardinality constraint on Procedure.performed[x], changing it from 1..1 to 0..1 [FHIR-45109](https://jira.hl7.org/browse/FHIR-45109)
+  - removed incorrect Obligation on Procedure.bodySite [FHIR-46728](https://jira.hl7.org/browse/FHIR-46728)
```

Comment:
change 'incorrect Obligation' to 'Obligations'. Apply this change to the two other instances.

Reply:
Amended.